### DISCUSSION TITLE
Pimcore 11 version dataobject issue

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [pimcore/pimcore#16506] [General]
### Pimcore version

11.1

### Steps to reproduce

Hi Team,

After successfully migrated and executed commands as per document, still i have data object issue.

### Actual Behavior

i noticed, in the classes table, few of the columns are migrated, few are not, still objects table has o_versionCount, thats causes issue, how to migrate specific table columns to get it resolve issues?

few classification store, collection table has o prefixes, how to migrate it?
![classificationStore](https://github.com/pimcore/pimcore/assets/78895325/0076e2f8-0d20-4d0e-9ea5-b2cb57644f64)

Please reply for this issue to resolve?
correctly not migrated all the tables , few are migrated and few are not?

### Expected Behavior

It should work

**Comment by kingjia90** [comment]
This was covered by https://github.com/pimcore/pimcore/blob/40890a3c30481fec0d15dbd0b550b1865a55d8fc/bundles/CoreBundle/src/Migrations/Version20221003115124.php#L60-L67 

Please try running the migration

**Comment by Thejaswini-Rao-U** [comment]
@kingjia90 , I tried this but im getting below error on foreign key, bcz it doesnt migrate o_id alone?

![foreignKeyObjectIssue](https://github.com/pimcore/pimcore/assets/78895325/f9d20aec-b80c-45d8-90a2-9a3993ba2303)

what is the solution solve this? how to overcome foreign key issue?

**Comment by kingjia90** [comment]
Maybe trying by adding `$this->addSql('SET foreign_key_checks = 0');` on the first line of `up()`

**Comment by Thejaswini-Rao-U** [comment]
@kingjia90 , i did, but it didnt work, still same error

**Comment by Thejaswini-Rao-U** [comment]
Hi
Please reply

**Comment by kingjia90** [comment]
Check if the mentioned fields in the error are matching.
I see a BIGINT in the screenshot, while the migration changes to 'int(11) unsigned'

See also https://stackoverflow.com/questions/21526055/mysql-cannot-create-foreign-key-constraint

**Reply by kingjia90** [reply]
Moved to discussion area, it was changed since 
https://github.com/pimcore/pimcore/commit/f48d52ee4564cacd8b5dfe58414de626babbd419#diff-5959ee96b2dc092ba3e0a5d9834745486efc70ed786934bd466e72e310a575f7R236 and fixed by https://github.com/pimcore/pimcore/pull/10755

**Reply by Thejaswini-Rao-U** [reply]
@kingjia90  Thanks, will check
